code/modules/projectiles/ammunition/ballistic/lmg.dm

```diff
@@ -1,23 +1,23 @@
-// 1.95x129mm (SAW)
+// 5.56x28mm (SAW)
```

Comment:
Arbitrary and unneeded change.

Reply:
5.56x28mm lmao what

current standard rifle round of 2020 is 5.56x45mm, what you made is now a pistol caliber reminiscent of 5.7x28mm. This is an LMG  just keep the caliber although 1.95x129mm is ludicrous aswell but atleast it's a rifle caliber.

Reply:
Fair. Reverted